lib/index.ts

```diff
@@ -163,6 +164,7 @@ function parseAnalysisResults(
     type: depType,
     depInfosList: analysisResult.Analysis,
     binaries: analysisJson.binaries.Analysis,
+    hashes: analysisJson.hashes.Analysis,
```

Comment:
Maybe we should rename it to binariesHashes?
We'll use this value in other places in the code (not only the plugin) and it should be clear what's the content of it

Reply:
I am not sure why binary hashes is more clear? Can hashes be anything else than binary? I use the shorter terminology throughout the code so would like to keep the naming consistent.

Reply:
think about it that at the end we'll have `hashes` section inside the DB - we need to make sure that people will understand what's the meaning of those hashes - so the trivial question for me (without any context) will be "hashes of what?"

Reply:
Not sure what you mean by "inside the DB". I would then add a comment to make it clear. Naming it binariesHashes I would think that these are the actual hash values of the objects in the binaries variable. Trying to convey a description with a variable name will be confusing either way.